test/e2e/indexTable/indexTable.spec.ts

```diff
@@ -102,6 +102,7 @@ test('IndexTable - empty state', async ({ page }) => {
 test('IndexTable - scrollable', async ({ page }) => {
   const indexTablePage = new IndexTablePage(page);
 
+  page.setViewportSize({ width: 500, height: 500 });
```

Comment:
just wondering why you need to set such width and height for this test?

Reply:
To make it obvious it's scrollable